src/main/java/io/cdap/delta/bigquery/BigQueryAssessor.java

```diff
@@ -29,13 +28,18 @@
 
 import java.util.ArrayList;
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.Objects;
 
 /**
  * Assesses table information.
  */
 public class BigQueryAssessor implements TableAssessor<StandardizedTableDetail> {
   private final String stagingTablePrefix;
+  // tables already assessed so far, key is table
```

Comment:
nit: add doc about what is value too or better rename variable name to something like `tableSchema`. Also better initialize in constructor at one place.

Meaning 
private final Map<String, String> tableSchema;

and in constructor do
tableSchema = new HashMap<>();

Reply:
done :)